src/library/core.ts

```diff
@@ -52,7 +52,8 @@ export default class Core {
 
     if (defaultChannel && defaultChannel instanceof TextChannel) {
       defaultChannel.send(`'${username}' has joined this server`);
-      defaultChannel.send(`Welcome, <@${id}> !`);
+      defaultChannel.send(`Welcome, <@${id}> ! Please request a role with the command !role.
+      Use !role help for more information.`);

```

Comment:
Note to make sure it line breaks as you expect it to, it may just put a lot of spaces instead. You might need to explicitly add line break characters. (I think those come through in template literals though... 🤔 You might still have a lot of spaces though ¯\\\_(ツ)\_/¯